src/main/kotlin/org/jetbrains/research/libsl/nodes/annotations.kt

```diff
@@ -31,12 +31,12 @@ data class DeclaredAnnotation(
     override fun toString(): String = dumpToString()
 
     override fun dumpToString(): String = buildString {
-        append("annotation ${BackticksPolitics.forPeriodSeparated(name)}(")
+        append("annotation ${BackticksPolitics.forPeriodSeparated(name)}")
         if(values?.isNotEmpty() == true) {
-            appendLine()
+            appendLine("(")
             appendLine(withIndent(values.joinToString(separator = ",\n" ) { value -> value.dumpToString() }))
+            append(")")
```

Comment:
This code adds an closing `)` at the line of the last parameter. I'd add a line break before it:
```
@Anno(
    param1,
    param2
)
```